crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -196,6 +201,30 @@ fn get_private_key_from_input() -> Result<Felt> {
     parse_input_to_felt(&input)
 }
 
+fn generate_account_name(accounts_file: &Utf8PathBuf) -> anyhow::Result<String> {
+    let networks: NestedMap<AccountData> = read_and_parse_json_file(accounts_file)?;
+    let mut result = HashSet::new();
+
+    for (_network, accounts) in networks {
+        for (name, _data) in accounts {
+            if let Some(id) = name
+                .strip_prefix("account-")
+                .and_then(|id| id.parse::<u32>().ok())
+            {
+                result.insert(id);
+            };
+        }
+    }
+
+    let mut id = 1;
+
+    while result.contains(&id) {
+        id += 1;
+    }
+
+    Ok(format!("account-{id}"))
+}
+
```

Comment:
The logic here is good, it just needs to handle the case when there is no json file with the accounts. In such case, `read_and_parse_json_file` will return `Err`. We should handle it and return `Ok(format!("account-{id}"))` (`id` should be equal to `1` at that time, because it will be first imported account, obvious).

It could look like this:
```rust
fn generate_account_name(accounts_file: &Utf8PathBuf) -> anyhow::Result<String> {
    let mut id: u32 = 1;
    let networks: NestedMap<AccountData> = match read_and_parse_json_file(accounts_file) {
        Ok(networks) => networks,
        Err(_) => return Ok(format!("account-{id}")),
    };
    
    let mut result = HashSet::new();

    for (_network, accounts) in networks {
        for (name, _data) in accounts {
            if let Some(id) = name
                .strip_prefix("account-")
                .and_then(|id| id.parse::<u32>().ok())
            {
                result.insert(id);
            };
        }
    }

    while result.contains(&id) {
        id += 1;
    }

    Ok(format!("account-{id}"))
}
```